### ISSUE TITLE
New light-animation commands for FW e.g. running-light

### ISSUE THREAD

**Issue by polybassa** [polybassa/WyLight#56]

**Comment by polybassa** [comment]
Idea: Implement a gradient command. 
commandstruct: CMD COLORLEFT COLORRIGHT FADETMMS PARALLEL

function: we can save this command in our scriptbuffer, too. If the command is executed it set's the destination colors to a gradient form the left to the right color and starts to fade to these colors.

**Comment by bruenn** [comment]
Good idea but add a range to the command if we have products with less 
than 32 leds. Two byte, one for the start and one the endindex, should be sufficient
